src/emulator/auth/state.ts

```diff
@@ -551,6 +552,87 @@ export class ProjectState {
     }
   }
 }
+
+export class AgentProjectState extends ProjectState {
+  private tenantForTenantId: Map<string, Tenant> = new Map();
+
+  constructor(
+    projectId: string,
+    private _oneAccountPerEmail = true,
```

Comment:
Those things don't make sense in constructor since they are never injected in. Consider making them simply just properties (like `tenantForTenantId`)

Reply:
Done